Sum_of_N_Natural_Numbers/Sum_of_N_Natural_Numbers.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+    Sum of First N Natural Numbers
+
+    The task is to find sum of first n natural numbers, where n is 
+    the number input to us. We use the well known formula,
+    sum = n * (n + 1)/2 to compute the sum
+
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int main() {
+    int num;
+    cout << "Enter N: ";
+    cin >> num;
```

Comment:
There is no need to give any space in the cout & cin statements. The compiler just ignores unnecessary whitespaces. Approving this for now.